charts/hono/templates/hono-adapter-lora/hono-adapter-lora-vertx-secret.yaml

```diff
@@ -34,6 +34,7 @@ stringData:
         {{- end }}
       {{- include "hono.healthServerConfig" .Values.adapters.lora.hono.healthCheck | nindent 6 }}
       {{- include "hono.serviceClientConfig" $args | nindent 6 }}
+    {{- include "hono.quarkusConfig" $args | indent 4 }}
```

Comment:
It looks like I had forgotten to add this to the secret definitions of the CoAP and AMQP adapter. Can you add it as part of this PR (maybe as a separate commit) as well?

Reply:
Done